core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/ViewNames.java

```diff
@@ -31,6 +31,7 @@
     public static final String POS = "POS";
 
     public static final String MENTION = "MENTION";
+    public static final String RELATION = "RELATION";
```

Comment:
Please add a corresponding switch case in the 'getViewType()' method (https://github.com/danyaljj/illinois-cogcomp-nlp-1/blob/60b45aadcd689fbe5fc6e0af66ca0aa585c0cff7/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/ViewNames.java#L134)

Reply:
Done.